<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="apples-MacBook-Pro.local" name="com.zaarolbasic.testCases.TC_LoginTest_001" tests="7" failures="0" timestamp="24 Nov 2021 11:11:42 GMT" time="31.771" errors="1">
  <testcase name="loginTest_validEmail_validPassword_TC_001" time="26.146" classname="com.zaarolbasic.testCases.TC_LoginTest_001"/>
  <testcase name="loginTest_invalidEmail_validPassword_TC_002" time="1.241" classname="com.zaarolbasic.testCases.TC_LoginTest_001"/>
  <testcase name="loginTest_invalidEmail_invalidPassword_TC_003" time="0.837" classname="com.zaarolbasic.testCases.TC_LoginTest_001">
    <error type="java.lang.IllegalArgumentException" message="Keys to send should be a not null CharSequence">
      <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:97)
at com.zaarolbasic.pageObjects.LoginPage.setPassword(LoginPage.java:140)
at com.zaarolbasic.testCases.TC_LoginTest_001.loginTest_invalidEmail_invalidPassword_TC_003(TC_LoginTest_001.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- loginTest_invalidEmail_invalidPassword_TC_003 -->
  <testcase name="loginTest_validEmail_EmptyPassword_TC_004" time="0.963" classname="com.zaarolbasic.testCases.TC_LoginTest_001"/>
  <testcase name="loginTest_invalidEmail_EmptyPassword_TC_005" time="0.929" classname="com.zaarolbasic.testCases.TC_LoginTest_001"/>
  <testcase name="loginTest_EmptyUsername_InvalidPassword_TC_006" time="0.800" classname="com.zaarolbasic.testCases.TC_LoginTest_001"/>
  <testcase name="loginTest_EmptyUsername_EmptyPassword_TC_007" time="0.855" classname="com.zaarolbasic.testCases.TC_LoginTest_001"/>
</testsuite> <!-- com.zaarolbasic.testCases.TC_LoginTest_001 -->
